src/fs/file_format.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ * @brief Detect file format by header/etc
+ * @author Denis Deryugin <deryugin.denis@gmail.com>
+ * @version 0.1
+ * @date 2015-12-10
+ */
+
+#include <fs/file_format.h>
+
+#include <string.h>
+#include <unistd.h>
+
+enum fformat raw_get_file_format(uint8_t *file) {
+	/* Is it a RIFF file? */
+	if (!memcmp(file, "RIFF", 4)) {
+		/* TODO check file consistency */
+		return RIFF_FILE;
+	}
+
+	return TEXT_FILE; /* Regular text file */
```

Comment:
May be, REGULAR_FILE? 
as for me text file is associated with not binary format file